eng/versioning/version_client.txt

```diff
@@ -194,6 +194,7 @@ com.azure.resourcemanager:azure-resourcemanager-loganalytics;1.0.0-beta.1;1.0.0-
 com.azure.resourcemanager:azure-resourcemanager-eventgrid;1.0.0-beta.1;1.0.0-beta.2
 com.azure.resourcemanager:azure-resourcemanager-healthbot;1.0.0-beta.1;1.0.0-beta.1
 com.azure.resourcemanager:azure-resourcemanager-confluent;1.0.0-beta.1;1.0.0-beta.2
+com.azure.quantum.jobs:azure-quantum-jobs;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
This list of library versions should be alphabetically sorted.

Reply:
Done